resources/assets/coffee/react/status.coffee

```diff
@@ -0,0 +1,272 @@
+          div className: 'status__recent-incidents osu-layout__col osu-layout__col--sm-6 osu-layout__row--page-compact',
+            h1 null,
+              Lang.get('statusPage.recent.incidents.title')
+            div className: 'status__recent-incidents-when',
+              h5
+                onClick: @_changeViewMode.bind(@, 'incidents', 'today')
+                className: 'status__recent-incidents--active' unless @state.incidents != 'today'
+                Lang.get('statusPage.recent.when.today')
+              h5 
+                onClick: @_changeViewMode.bind(@, 'incidents', 'last_week')
+                className: 'status__recent-incidents--active' unless @state.incidents != 'last_week'
+                Lang.get('statusPage.recent.when.last_week')
+              h5 
+                onClick: @_changeViewMode.bind(@, 'incidents', '2_weeks')
+                className: 'status__recent-incidents--active' unless @state.incidents != '2_weeks'
+                Lang.choice('statusPage.recent.when.weeks_ago', 2)
+              h5 
+                onClick: @_changeViewMode.bind(@, 'incidents', '3_weeks')
+                className: 'status__recent-incidents--active' unless @state.incidents != '3_weeks'
+                Lang.choice('statusPage.recent.when.weeks_ago', 3)
+            div null,
+              incidents
+          div className: 'status__recent-uptime osu-layout__col osu-layout__col--sm-6 osu-layout__row--page-compact',
+            h1 null,
+              Lang.get('statusPage.recent.uptime.title')
+            div className: 'status__recent-uptime-when',
+              h5
+                onClick: @_changeViewMode.bind(@, 'uptime', 'today')
+                className: 'status__recent-uptime--active' unless @state.uptime != 'today'
+                Lang.get('statusPage.recent.when.today')
+              h5 
+                onClick: @_changeViewMode.bind(@, 'uptime', 'week')
+                className: 'status__recent-uptime--active' unless @state.uptime != 'week'
+                Lang.get('statusPage.recent.when.week')
+              h5 
+                onClick: @_changeViewMode.bind(@, 'uptime', 'month')
+                className: 'status__recent-uptime--active' unless @state.uptime != 'month'
+                Lang.get('statusPage.recent.when.month')
+              h5 
+                onClick: @_changeViewMode.bind(@, 'uptime', 'all_time')
+                className: 'status__recent-uptime--active' unless @state.uptime != 'all_time'
+                Lang.get('statusPage.recent.when.all_time')
+            div className: 'status__recent-charts osu-layout__col-container',
+              div
+                className: 'osu-layout__col osu-layout__col--sm-6'
+                ref: 'serverChart'
+              div
+                className: 'osu-layout__col osu-layout__col--sm-6'
+                ref: 'webChart'
+
+ReactDOM.render el(Status.Page), document.getElementsByClassName('js-content')[0]

```

Comment:
this shouldn't be here. Check `resources/assets/coffee/react/profile-page.coffee`